crates/autopilot/src/solvable_orders.rs

```diff
@@ -293,6 +308,20 @@ async fn filter_banned_user_orders(
     orders
 }
 
+fn get_native_prices(
+    tokens: &[H160],
+    native_price_estimator: &CachingNativePriceEstimator,
+) -> HashMap<H160, U256> {
+    native_price_estimator
+        .get_cached_prices(tokens)
+        .into_iter()
+        .flat_map(|(token, result)| {
+            let price = to_normalized_price(result.ok()?)?;
```

Comment:
We silently ignore this, I know it was there from before, but maybe it is worth adding a warning here :thinking: